docs/source/_static/style.css

```diff
@@ -0,0 +1,3 @@
+.wy-nav-content {
+    max-width: 1200px !important;
```

Comment:
@tomvothecoder The width of the docs on https://e3sm-project.github.io/e3sm_data_docs/_build/html/simulation_locations.html is pretty narrow -- much of the table is obscured.

I seem to recall needing to do something on `zppy`'s docs to widen the page. I tried adding https://github.com/E3SM-Project/zppy/blob/main/docs/source/_static/style.css#L2 but this doesn't seem to widen the page.